package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "react-date-time-range",
+  "version": "1.0.0",
+  "description": "",
+  "main": "dist/DateTimeRange.js",
+  "directories": {
+    "doc": "doc",
+    "test": "test"
+  },
+  "scripts": {
+    "ci": "npm run lint && npm run coverage",
+    "coverage": "istanbul cover _mocha -- --compilers js:babel/register --require react-tests-globals-setup test/test* && istanbul check-coverage",
+    "prepublish": "mkdir -p dist && babel src/DateTimeRange.jsx > dist/DateTimeRange.js",
+    "prestart": "ulimit -n 9999",
+    "start": "watchify -t babelify -t brfs doc/example.js -o doc/example-built -v",
```

Comment:
we could use a package like [serve](https://www.npmjs.com/package/serve) to start a simple dev server instead of relying on rendering html from the file system